Configuration/StandardSequences/python/DigiToRawPreMixing_cff.py

```diff
@@ -43,10 +43,10 @@
 
 # GEM settings
 _run2_GEM_2017_DigiToRaw = DigiToRaw.copy()
-_run2_GEM_2017_DigiToRaw.insert(0,gempacker)
+_run2_GEM_2017_DigiToRaw.insert(-1,gempacker)
 
 _run3_DigiToRaw = _run2_GEM_2017_DigiToRaw.copy()
-#_run3_DigiToRaw.insert(0,me0packer) # to be done once proper ME0 digitiser is done
+_run3_DigiToRaw.insert(-1,me0packer)
```

Comment:
ME0 should be enabled in Phase-2, not Run-3. This explains the crash in WF 11624.0.